src/components/scenes/SendScene.js

```diff
@@ -362,7 +359,8 @@ class SendComponent extends React.PureComponent<Props, State> {
   }
 
   renderAmount() {
-    const { exchangeRates, lockInputs, lockTilesMap = {}, hiddenTilesMap = {}, nativeAmount, settings, theme } = this.props
+    const { exchangeRates, lockInputs, nativeAmount, settings, theme } = this.props
```

Comment:
`const { ... other stuff ... , route } = this.props`, and then you can use `route` on the next line without the `this.props.` bit.